string DATA
string POSE
---
string IDS